### DISCUSSION TITLE
Docs for extensions

### DISCUSSION THREAD

**Discussion by Attacler** [directus/directus#5256] [Feature Requests]
Can we have a propper documentation on extensions.
For example with the module extension, how to populate the left/right bar:
![image](https://user-images.githubusercontent.com/42644889/116092304-fa0e7c00-a6a5-11eb-94d1-bed252d58081.png)
This will probbaly help out a lot of people!

**Comment by benhaynes** [comment]
Yes, this would be good to explain and provide examples for. @rijkvanzanten @nickrum can one of you add some quick usage examples to the Docs? 🙏

**Comment by DaanMoura** [comment]
I agree that the documentation should be improved. Sometimes I look into the code of the built-in modules of Directus.
There are templates to populate those bars:

```vue
<template>
    <private-view>
        <template #navigation>
            Left bar
        </template>
        <template #sidebar>
            Right bar
        </template>
    </private-view>
</template>
```

**Comment by jaads** [comment]
Heya! Thank you for taking the time to submit this request! 

It has been over **90 days**, and this discussion has not received at least **15 votes** from the community. This means that we don't feel like there's enough community interest to warrant further R&D into this topic at this time. :ice_cube:

This request will now be closed to keep our discussions tidy. Please reach out if you have any questions!

_For more information, see our [Feature Request Process](https://docs.directus.io/contributing/feature-request-process.html)._
